# top left corner
float64 x1
float64 y1
# system right-down
# bottom right corner
float64 x2
float64 y2
